File: master-package/bonus-webinars-2018/running-all-tests-with-maven/src/integration-test/java/com/testwithspring/master/IntegrationTest.java
package com.testwithspring.master;

import org.junit.jupiter.api.DisplayName;
import org.junit.jupiter.api.Tag;
import org.junit.jupiter.api.Test;

@Tag(TestTags.INTEGRATION_TEST)
class IntegrationTest {

    @Test
    @DisplayName("This test method should be run when we run our integration tests")
    void shouldBeRunWhenIntegrationTestsAreRun() {
    }
}

File: master-package/bonus-webinars-2019/introduction-to-mockk/src/integration-test/kotlin/com/testwithspring/master/kotlin/Integrationtest.kt
package com.testwithspring.master.kotlin

import org.assertj.core.api.Assertions
import org.junit.jupiter.api.BeforeEach
import org.junit.jupiter.api.DisplayName
import org.junit.jupiter.api.Tag
import org.junit.jupiter.api.Test

@Tag("integrationTest")
class Integrationtest {

    @Test
    @DisplayName("Should be when we run our integration tests")
    fun shouldBeRunWhenWeRunOurIntegrationTests() {
        println("Should be run when we run our integration tests")
    }
}